tests/src/test/scala/apigw/healthtests/ApiGwEndToEndTests.scala

```diff
@@ -166,6 +170,7 @@ abstract class ApiGwEndToEndTests
         basepathOrApiName = testbasepath,
         expectedExitCode = DONTCARE_EXIT,
         cliCfgFile = Some(cliWskPropsFile.getCanonicalPath()))
+      println("api removed prior to recreation: " + testbasepath)
```

Comment:
these prints stay?

Reply:
no.. these were just temporary adds to troubleshoot and issue that was eventually resolved with a recalibrated database.  i'll remove those new printlns

Reply:
thx